gatsby-node.js

```diff
@@ -33,6 +34,19 @@ exports.createPages = ({ actions, graphql }) => {
 
     posts.forEach(edge => {
       const id = edge.node.id
+      if(edge.node.frontmatter.path) {
```

Comment:
The difference between the two being only the `path` field, the if/else should be used just to assign the appropriate path to a variable. The `createPage` call needs to happen only once. #DRY